### PR TITLE
Production Builds/Migrate to Bootstrap

### PR DESCRIPTION
**Pull request by larkmullins**

### CODE REVIEW

**Comment by kyleferguson** [comment]
@larkmullins if it works it's good with me. Need to fix conflicts.

**Comment by kyleferguson** [comment]
Need to get these changes and stable production builds done ASAP. If you need help let me know.

**Review by kyleferguson — APPROVED** [review]
Great work, LGTM!

File: ui/src/app/masthead/masthead.component.ts [hunk 1]
```diff
@@ -17,19 +16,21 @@ export class MastheadComponent implements OnInit {
...
-  public modalActions: EventEmitter<MaterializeAction>;
+  public modalActions: EventEmitter<string>;
 
   constructor(private router: Router, private applicationService: ApplicationService, private _fb: FormBuilder) {
-    this.modalActions = new EventEmitter<string|MaterializeAction>();
+    this.modalActions = new EventEmitter<string>();
   }
-
+  
+  /*
```
Comment (jlaswell) [inline ui/src/app/masthead/masthead.component.ts]: If this is just a comment, I say we can rip it out.

File: ui/src/app/resources/applications/application-details.component.ts [hunk 2]
```diff
@@ -20,7 +19,7 @@ export class ApplicationComponent implements OnInit {
   public createEnvironmentForm: FormGroup;
   private environments: Environment[];
   public submitted: boolean;
-  private modalActions = new EventEmitter<string|MaterializeAction>();
+  //private modalActions = new EventEmitter<string>();
```
Comment (jlaswell) [inline ui/src/app/resources/applications/application-details.component.ts]: If this is just a comment, I say we can rip it out.

**Comment by jlaswell** [comment]
Nice work here!

**Pull request merged**